Classes/Command/QueueCommandController.php

```diff
@@ -40,14 +42,15 @@ class QueueCommandController extends CommandController
      * Displays all configured queues, their type and the number of messages that are ready to be processed.
      *
      * @return void
+     * @throws Exception
```

Comment:
Our CGL don't state anything about the requirement of `@throws` annotations. Personally I'm strongly [against those](https://discuss.neos.io/t/to-throws-or-not-to-throws-indirect-exceptions/3018/4).
Obviously no reason to block this, but if we decide to add the annotations we should do it for the whole package, and maybe in a separate PR